index.html

```diff
@@ -30,14 +30,16 @@ <h2>GitHub for the Technical Communications Classroom</h2>
 
 <div class="textblock" style="text-align: left">
 <h1>Participant Code Narratives</h1>
-  <p>Hello! I'm Steve and I work at the University of Pittsburgh. I have always been scared of code...mainly, I think, because I lacked exposure to it. While I did use a little HTML when I ran my own business, in my teaching and practice I have relied on visual design web tools like Adobe Muse, Webflow, and drag-n-drop tools like Mobirise, Wix, and Weebly. But at some point I realized that the more I work with and these tools, the more I am at the mercy of corporations and their code. As a result, I have been working to develop tools and pedagogies to help introduce educators and students to code, computer science, and open-source collaboration.</p>
+  <p>Hello! I'm Stevo.</p>
   </div>
 <br>
 <br>
 <br>
 <br>
 
 
```

Comment:
&lt;div class="textblock" style="text-align: left"&gt;&lt;h1&gt;Participant Code Narratives&lt;/h1&gt;  &lt;p&gt;Hello! I'm Steve and I work at the University of Pittsburgh. I have always been scared of code...mainly, I think, because I lacked exposure to it. While I did use a little HTML when I ran my own business, in my teaching and practice I have relied on visual design web tools like Adobe Muse, Webflow, and drag-n-drop tools like Mobirise, Wix, and Weebly. But at some point I realized that the more I work with and these tools, the more I am at the mercy of corporations and their code. As a result, I have been working to develop tools and pedagogies to help introduce educators and students to code, computer science, and open-source collaboration.&lt;/p&gt;  &lt;p&gt;Hello! I'm Stevo.&lt;/p&gt;  &lt;/div&gt;&lt;br&gt;&lt;br&gt;&lt;br&gt;&lt;br&gt;